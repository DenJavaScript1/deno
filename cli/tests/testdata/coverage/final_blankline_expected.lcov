SF:[WILDCARD]final_blankline.ts
FN:2,example
FNDA:1,example
FNF:1
FNH:1
BRF:0
BRH:0
DA:2,1
DA:3,2
DA:4,2
LH:3
LF:3
end_of_record
