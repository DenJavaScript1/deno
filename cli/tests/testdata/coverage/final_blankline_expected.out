cover file:///[WILDCARD]/final_blankline.ts ... 100.000% (3/3)
